_data/api-swagger.yaml

```diff
@@ -451,6 +451,7 @@ definitions:
           - eIDAS Login
           - iDeal
           - iDIN
+          - Itsme Identification
```

Comment:
Please use the same casing as the manufacturer / supplier uses. In this case `itsme`